6.analyze_features/scripts/0.aggregate_bones.py

```diff
@@ -123,7 +133,7 @@
 custom_aggregated_data = pd.DataFrame()
 
 # define the metadata columns to aggregate by
-metadata_cols = ["Metadata_genotype", "Metadata_replicate"]
+metadata_cols = ["Metadata_genotype", "Metadata_replicate", "Metadata_side"]
```

Comment:
Probably ok for now, but to reduce the potential for bugs (and in future projects) probably should define this list first, and then use it whenever you need to reference it (i.e., all the other changes in this notebook)